src/index.ts

```diff
@@ -1,9 +1,44 @@
-// tslint:disable:no-console
-
+import * as ddTracer from "dd-trace"
```

Comment:
Maybe move this line below the `dotenv/config`? That way we can be sure that the first thing being set is our ENV variables.

Reply:
Yeah that makes sense.